CHANGELOG.md

```diff
@@ -9,11 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- An issue where the connection between ASB and CLI would get closed prematurely.
-  The CLI expects to be connected to the ASB throughout the entire swap and hence reconnects as soon as the connection is closed.
-  This resulted in a loop of connections being established but instantly closed again because the ASB deemed the connection to not be necessary.
+- An issue where the connection between ASB and CLI would get closed prematurely. The CLI expects to be connected to the
+  ASB throughout the entire swap and hence reconnects as soon as the connection is closed. This resulted in a loop of
+  connections being established but instantly closed again because the ASB deemed the connection to not be necessary.
   See issue https://github.com/comit-network/xmr-btc-swap/issues/648.
 
+- An issue where swap protocol was getting stuck trying to submit the cancel transaction. We were not handling the error
+  when TxCancel submission fails. See issues: https://github.com/comit-network/xmr-btc-swap/issues/709
+  , https://github.com/comit-network/xmr-btc-swap/issues/688, https://github.com/comit-network/xmr-btc-swap/issues/701
```

Comment:
Not critical but these formatting changes seem to be unnecessary :)

Reply:
hmmm wonder how this happened

Reply:
Let's fix it in a follow-up. Would be good to get a release out with this fix!